packages/slate-editor/src/modules/editor-v4-rich-formatting/components/RichFormattingMenu.tsx

```diff
@@ -30,11 +31,18 @@ import { ElementType } from '../types';
 import { RichFormattingToolbar } from './RichFormattingToolbar';
 
 interface Props {
+    alignmentControls: boolean;
     containerRef: RefObject<HTMLElement>;
+    defaultAlignment: Alignment;
     parameters: RichFormattingExtensionParameters;
 }
 
-export const RichFormattingMenu: FunctionComponent<Props> = ({ containerRef, parameters }) => {
+export const RichFormattingMenu: FunctionComponent<Props> = ({
```

Comment:
nit. FC or FunctionalComponent have some disadvantages https://github.com/facebook/create-react-app/pull/8177

Reply:
Interesting.

So do you think we should stop using `FunctionalComponent` for defining components?

Reply:
in my opinion yes, we could use `function Component(props: Props)` or `function Component(props: React.PropsWithChildren<Props>)`

Reply:
I actually love it! Will try to adopt this format for a few new components before deciding to go all-in. Thanks!